testbed.md

```diff
@@ -340,7 +347,7 @@ With figure components
 ### Grid of citations
 
 {% capture content %}
-{% include list.html data="citations" component="citation" %}
+{% include list.html data="citations" component="citation" style="rich" %}
```

Comment:
Would it make sense to show both the `rich` and non-`rich` versions of the citations sequentially here for quick comparison of testing or appearance?

Reply:
I show rich and non-rich in the dedicated citations component section above. The rich style is only important to test in the grid because of the new css container queries I'm using here.